mkosi

```diff
@@ -3793,6 +3796,8 @@ def create_parser() -> ArgumentParserMkosi:
                        help='Script to call instead of mksquashfs')
     group.add_argument("--xz", action=BooleanAction,
                        help='Compress resulting image with xz (only gpt_ext4, gpt_xfs, gpt_btrfs, gpt_squashfs, implied on tar)')  # NOQA: E501
+    group.add_argument("--zstd", action=BooleanAction,
+                       help='Compress resulting image with zstd (only gpt_ext4, gpt_xfs, gpt_btrfs, gpt_squashfs, implied on tar. Conflicts with --xz option.)')
```

Comment:
"implied on tar" should be removed here as well.

Reply:
removed from --zstd